### DISCUSSION TITLE
Internet Radio.

### DISCUSSION THREAD

**Discussion by mikronwfh** [pschatzmann/ESP32-A2DP#441] [Q&A] [answered]
How do we make internet radio out of what we have here?

**Comment by pschatzmann** [comment, marked as answer]
I think you are in the wrong project. 
I suggest to have a look at this: https://github.com/pschatzmann/arduino-audio-tools

**Comment by renatoianhez** [comment]
Maybe my project can help you: [https://github.com/renatoianhez/Lolin32-radio](https://github.com/renatoianhez/Lolin32-radio)
